app.js

```diff
@@ -1,5 +1,77 @@
 
 // state
+function getDefaultVolume() {
+    
+    const defaultVolume = {
+        level: 10,
+    };
+    return defaultVolume;
+    
+}
+
+let volume = getDefaultVolume();
+let click = false;
+
+
+
+//variables
+const controlsSection = document.getElementById('controls-section');
+
+const [downButton, upButton] = controlsSection.querySelectorAll('button');
+let volumeDisplay = controlsSection.querySelector('span');
```

Comment:
The dom selection looks good! you've referenced the two buttons and the volume display